README.md

```diff
@@ -0,0 +1,130 @@
+# Tahoe-LAFS Infrastructure
+
+The software that defines the Tahoe-LAFS project's infrastructure.
+
+## Abstract
+
+This repository was initially created in Sep 2022 as an effort to deploy and
+manage a `testgrid` server using NixOS (now retired).
+
+Since January 2025, some members of the community are seeking to re-use this repository
+to deploy and manage some new systems:
+
+- a `testgrid` server re-using the NixOS configuration written in 2022
```

Comment:
This should link to https://tahoe-lafs.org/trac/tahoe-lafs/wiki/TestGrid

Reply:
Good idea: I've added the link in the Testgrid section.